app/src/types/ChefsSubmissionForm.ts

```diff
@@ -35,4 +36,9 @@ export type ChefsSubmissionForm = {
   user: User | null; // assigned to
   intakeStatus: string | null;
   applicationStatus: string | null;
+  guidance: boolean | null;
+  statusRequest: boolean | null;
+  inquiry: boolean | null;
+  emergencyAssist: boolean | null;
+  inapplicable: boolean | null;
```

Comment:
You made these non nullable in the schema so the app type should not be nullable.